src/__tests__/queries.find.js

```diff
@@ -0,0 +1,119 @@
+import {render} from './helpers/test-utils'
+
+test('find asynchronously finds elements', async () => {
+  const {
```

Comment:
would it be helpful to add some asynchronicity to the tests, or is that covered by unit tests for waitForElement?

```js
const {/* queries * /, rerender} = render(`<div />`)
setTimeout(() => rerender(`<div>correct dom</div>`), 100)
//  wait IRL if not using fake timers, or something like:
const findPromises = Promise.all([expect().toBeTruthy()])
jest.runAllTimers()
await findPromises
```

Reply:
I think I want to rely on `waitForElement` for these things. It's a pretty simple abstraction on top of `waitForElement` so I think we're probably good as they are.

That said, I'll go ahead and add one test just to be sure. Can't hurt.